action.yml

```diff
@@ -5,6 +5,14 @@ inputs:
   token:
     description: Required for permission to tag the repo.
     required: true
+  with_month:
```

Comment:
can you add these parameters to the readme?

Reply:
done